ci/pipelines/deploy-to-staging.yml

```diff
@@ -545,6 +559,9 @@ resource_types:
       tag: latest
 
 groups:
+  - name: adot
```

Comment:
This group needs to be later in the groups. The order of the groups is the order in the UI and for every other pipeline is:

* all node & java apps in alphabetical order; then
* all sidecars in alphabetical order

So it should appear after webhooks-egress  and before alpine